### PR TITLE
Update modelzoo

### PR DESCRIPTION
**Pull request by jin-s13**
add some checkpoints and logs.

### CODE REVIEW

File: configs/top_down/alexnet/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Imagenet classification with deep convolutional neural networks
```
Comment (innerlee) [inline configs/top_down/alexnet/README.md:1]: does top-down methods only differs by backbone?

Comment (jin-s13) [reply configs/top_down/alexnet/README.md:1]: Yes. For most top-down methods, the network architectures are the main differences.

Comment (innerlee) [reply configs/top_down/alexnet/README.md:1]: 🤔

Comment (innerlee) [reply configs/top_down/alexnet/README.md:1]: let's merge it as is, and change only when necessary

**Comment by innerlee** [comment]
Could you add the newly added models to the repo's readme?

**Comment by jin-s13** [comment]
> Could you add the newly added models to the repo's readme?

Sure.

**Pull request merged**